meinberlin/apps/livequestions/api.py

```diff
@@ -36,3 +39,9 @@ def get_queryset(self):
         ):
             live_questions = live_questions.filter(is_hidden=False)
         return live_questions
+
+    def dispatch(self, request, *args, **kwargs):
+        if request.method == 'POST':
+            body = json.loads(request.body.decode("utf-8"))
```

Comment:
I am bit unhappy with this, because I have to get the category manually, but didnt know how else to solve it..

Reply:
Maybe @rmader has an idea?

Reply:
Hm, given that it's not part of the livequestion model...do we by any chance have a mixin for the categories? Other than that I don't have an idea :/

Reply:
No, we don't have a category mixin. I could write one, but maybe thats overkill and we just leave it like that?

Reply:
Oh righ, it's ther from the weird a4-categorizable: https://github.com/liqd/a4-meinberlin/blob/166e44d2e947db6caf290afd187c44954f2602cb/meinberlin/config/settings/base.py#L474
So, I would say, leave it like that!